Library/Formula/vera++.rb

```diff
@@ -2,26 +2,21 @@
 
 class Veraxx < Formula
   homepage "https://bitbucket.org/verateam/vera"
-  url "https://bitbucket.org/verateam/vera/downloads/vera++-1.2.1.tar.gz"
-  sha1 "0bf7d463dabe41c2069dbe2fa4f7fca192cb7d6e"
+  url "https://bitbucket.org/verateam/vera/downloads/vera++-1.3.0.tar.gz"
+  sha1 "696c143e42df6854c9e5674b9e713fa8b9dae6f0"
 
-  depends_on "cmake" => :build
-  depends_on "boost"
-
-  # Use prebuilt docs to avoid need for pandoc
-  resource "doc" do
-    url "https://bitbucket.org/verateam/vera/downloads/vera++-1.2.1-doc.tar.gz"
```

Comment:
If it saves a dep it's probably worth using the prebuilt (unless our pandoc brings advantages)